docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -321,8 +321,8 @@ These guidelines are the policy for inclusion in the Ansible package and are in
 licensing and legal concerns that may otherwise affect your code.
 
 
-Contributor License Agreements
-==============================
+Contributor License Agreements (CLAs) and the DCO
```

Comment:
Fair enough. We can move this part to a later stage, perhaps add it to the inclusion criteria? @samccann is working on a project template that will include the DCO too, so that will help. I can remove this for now, I think.

Reply:
This section was likely added by @Andersson007 so just giving him a ping in case he has opinions.

Reply:
I would remove this from here and add the DCO files to https://github.com/ansible-collections/collection_template/ instead. All new collections are created from tat template.

Reply:
If we move it, lets move all of it. It doesn;t make sense to mention the prohibition of CLAs without mentioning the DCO. It's all or nothing, but I'm OK with nothing if it moves to the template.

Reply:
Your point about scope is well made, so I'll remove this for now. However, I do want to think about how we aim for consistency across the ecosystem - I'll follow that up with an SC forum topic once we've got this done.